assets/javascripts/modules/validatorFocus.js

```diff
@@ -7,10 +7,26 @@
     }
 
     $('.error-summary a').on('click', function(e) {
+      var $this = $(this);
+      var focusId = $this.attr('data-focuses');
+      if (!focusId)
+      {
+        focusId = ("" + $this.attr("href")).trim();
+        if (focusId.indexOf("#") !== 0)
```

Comment:
feels the same as `trim()` comment your guarding against developer error here. A developer including a link without a hash is it needed? and is there a genuine use case where a url without a hash maybe required?